Elements/SpacingConstants.cs

```diff
@@ -7,35 +7,31 @@ namespace Silksong.ModMenu.Elements;
 /// </summary>
 public static class SpacingConstants
 {
-    /// <summary>
-    /// About 1/4 the screen's width, making it good for 4-column setups.
-    /// </summary>
-    public const float HSPACE_SMALL = 10;
+    // TODO: Define HSPACE accessors based on aspect ratio
 
-    /// <summary>
-    /// About 1/3 the screen's width, making it good for 3-column setups.
-    /// </summary>
-    public const float HSPACE_MEDIUM = 20;
+    // About 1/4 the screen's width, making it good for 4-column setups.
+    // public static float HSPACE_SMALL => ?;
 
-    /// <summary>
-    /// About 1/2 the screen's width, making it good for 2-column setups.
-    /// </summary>
-    public const float HSPACE_LARGE = 30;
+    // About 1/3 the screen's width, making it good for 3-column setups.
+    // public static float HSPACE_MEDIUM => ??;
+
+    // About 1/2 the screen's width, making it good for 2-column setups.
+    // public static float HSPACE_LARGE => ???;
 
     /// <summary>
-    /// Spacing for about X small-font elements in a single column.
+    /// Spacing for about 12 small-font elements in a single column.
     /// </summary>
-    public const float VSPACE_SMALL = 70f;
+    public static float VSPACE_SMALL => 70f;
```

Comment:
Why the change from a const to a property? Consts are more efficient in general

Reply:
Consts can't do math on the screen aspect ratio, so making the change now for release compatibility later.